designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -255,8 +265,8 @@ fun SquooshRoot(
             else -> ""
         }
 
-    val layoutIdAllocator = remember { SquooshLayoutIdAllocator() }
-    val rootLayoutId = remember(docId) { SquooshLayout.getNextLayoutId() * 100000000 }
+    val layoutManager = remember(docId) { SquooshLayout.newLayoutManager() }
```

Comment:
This needs to move back up before line 241 so that `SquooshLayout.keepJniBits()` doesn't cause a crash because the layout manager can't be found.

Reply:
I don't see where the crash comes from -- I think the call to `keepJniBits` is a static method that just calls a JNI method with a bad handle (so the Rust side will log an error). I don't see a crash running Validation, but I could be missing something :/.

Reply:
Yea there will be a bad handle in Rust, and then it will throw an exception. This only happens if there is no cached `.dcf` file.
```
        throw_basic_exception(
            &mut env,
            &GenericError(format!("No manager with id {}", manager_id)),
        );
```

Reply:
Ok, I think I can probably delete `keepJniBits` entirely. I had it in because early on when working on Squoosh, the compiler would optimize away DC Classic, but since it's now a runtime thing whether we use Squoosh or not, DC Classic won't get optimized away.

Reply:
Ok, removed `keepJniBits`, will merge upon successful CI.